ONSdigital/response-operations-ui
# BUG: Changing case from a completed state to NOTSTARTED state in response ops

# What and why?
The functionality to turn a case to 'not started' when the status has been set to 'completed by phone' and 'no longer required' had been removed. This was required by the business users. 

The functionality has been added back in. The issues of  'not started' radio buttons appearing when the case was already in the 'not started' status has been resolved. They no longer show.

# How to test?
Change to completed by phone/no longer required:
Click reporting unit -> enter a reporting unit (I used 0001) -> Click search -> Click the number under RU reference -> Click the number under Survey -> Click 'Change' under the Status of the Collection Exercise table -> Click either 'Completed by phone' or 'No longer required' (these should be the only radio buttons) - > Confirm - > Redirect back to reporting unit page

Change back to not started:
Click the number under Survey -> Click 'View' under the Status of the Collection Exercise table -> Click 'Not started' - > Confirm

Testing can also be achieved by running the unit tests specific to this

# Trello
https://trello.com/c/sYxpWZPN/1898-bug-changing-case-from-a-completed-state-to-notstarted-state-in-response-ops-track-time
